Source/QuadTreeGrid/QuadTreeTemplateOperations.f90

```diff
@@ -53,7 +53,7 @@ SUBROUTINE ConstructNodesWithTemplate( self, rotation )
 !        Arguments
 !        ---------
 !
-         CLASS(QuadTreeGrid), POINTER :: self
+         TYPE (QuadTreeGrid), POINTER :: self
          INTEGER                   :: rotation ! Used for all other templates, otherwise ignored
```

Comment:
```suggestion
         TYPE (QuadTreeGrid), POINTER :: self
         INTEGER                      :: rotation ! Used for all other templates, otherwise ignored
```